app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None:
+    temp_lst = command.split()
+    if len(temp_lst) != 3 or any(value for value in temp_lst) is None:
```

Comment:
As per task requirement:

>     It must do nothing in case the user is trying to copy file to file with the same name.

So it would make sense to use a single if statement where you check if len is correct, if command is correct and if file name is not the same. That is much easier then making multiple ifs and raising exceptions in them. Also maybe you can use unpacking instead of accessing elements by index